src/data/finset/basic.lean

```diff
@@ -840,6 +840,14 @@ begin
     exact ⟨hk hbj, trans hti hk'⟩ }
 end
 
+lemma ite_subset_union (s s' : finset α) (P : Prop) [decidable P] :
```

Comment:
Do we already have `ite_le_sup` for `sup_order` (or whatever the right class is)? That would make this proof a one-liner.

Reply:
I can't find anything like that

Reply:
I think it would make sense to prove it in that generality first, since then we could add the lemmas to `set` too.